lib/archethic/p2p.ex

```diff
@@ -761,4 +750,11 @@ defmodule Archethic.P2P do
       conflict_resolver.(distinct_elems)
     end
   end
+
+  @doc """
+  Whether a node is synchronised or not is determined by whether the protocol chain node shared secrets are in sync.
+  When a new node renewal tx is produced, it is feasible that a node does not have that tx, resulting in an anomaly in election.
+  """
+  @spec verify_synchronization() :: :ok | :error | {:error, binary()}
+  def verify_synchronization(), do: SharedSecrets.is_synchronized(:node_shared_secrets)
```

Comment:
I don't get the purpose of this function because first it's on the P2P layer which handles the P2P communication, second it's just a delegation to SharedSecrets. I suggest in the Archethic module to directly check if the `SharedSecrets.checkSynchronization()` return `ok` before to send the tx, etc...

Reply:
idea wast to ask is this node synchronized? thats why it was in p2p